### ISSUE TITLE
Sorting on a multivalued field

### ISSUE THREAD

**Issue by awead** [projecthydra/hydra-collections#115]
With 8.0.0, title is now multivalued, but is also sortable:

https://github.com/projecthydra/hydra-collections/blob/master/app/models/concerns/hydra/collections/metadata.rb#L20

**Comment by jcoyne** [comment]
@awead I'm amazed too!

**Comment by jcoyne** [comment]
@awead apparently it's just picking one:
```ruby
Collection.new(title: ['one', 'two']).to_solr['title_si']
=> "two"
```

**Comment by awead** [comment]
@jcoyne interesting. This came up in our standup today because I was going through projecthydra/sufia#1614 and looking at the impact of a multivalued title. I don't think there's any way to guarantee that it will choose the first, or any numbered, title. Is there?

**Comment by jcoyne** [comment]
A bit deeper:
```ruby
Solrizer.set_field({}, 'sort_field', ['one', 'two'], :sortable)
=> {"sort_field_si"=>"two"}
```

**Comment by jcoyne** [comment]
@awead In CC, we only present one title on the form, so de-facto, we only have one title, so we don't have to worry about it.

**Comment by awead** [comment]
@jcoyne right, and it's the same behavior in Sufia, so it's probably a non-issue so long as title always has one value. The only way that might happen is if something gets created in console and not via the UI... so, very unlikely.

**Comment by awead** [comment]
Oh, but there's a validator...isn't there?

**Comment by jcoyne** [comment]
I don't know about any validator.  But here's the code where Solrizer handles sortables:

https://github.com/projecthydra/solrizer/blob/05e6a01b32553b9df434eacc35c0888f8bbc8050/lib/solrizer/field_mapper.rb#L228-L229

**Comment by awead** [comment]
@jcoyne I was referring to:
https://github.com/projecthydra-labs/curation_concerns/blob/master/curation_concerns-models/app/validators/has_one_title_validator.rb
but that doesn't seem to prevent a multiple title case, however, the UI excludes that.
